zrnn/datasets.py

```diff
@@ -2,15 +2,16 @@
 import torch
 from torch.utils.data import Dataset
 
-def generate_stimuli(period, T_onset, duration=2.0, dt=0.001, one_dur=0.01):
+def generate_stimuli(period, T_onset, duration=2.0, dt=0.001, one_dur=0.01, continuation_phase: bool = False):
```

Comment:
I'd say we should change the name of this keyword argument from `continuation_phase` to `I_stim_without_clicks` and flip the boolean logic.